### ISSUE TITLE
Frontend: endpoint `create-project/` is broken and return Error

### ISSUE THREAD

**Issue by kgevorkyan** [saveourtool/save-cloud#1082]
As noticed by @petertrr, probably because of
```kotlin
    useEffect {
        js(ENABLE_TOOLTIP_AND_POPOVER_JS)
        return@useEffect
    }
```
in InputForms.kt

**Comment by petertrr** [comment]
Yes, this is a consequence of using `ChildrenBuilder` extensions instead of components. We shouldn't be using hooks in extenstion functions to avoid such errors in the future, instead we should wrap them into `FC`s

**Comment by sanyavertolet** [comment]
Probably fixed by @akuleshov7 in #1089
